i18n/src/main/scala/io/udash/i18n/TranslationKey.scala

```diff
@@ -9,53 +10,55 @@ sealed trait TranslationKey {
   def key: String
 }
 
-sealed class TranslationKey0(override val key: String) extends TranslationKey {
+sealed case class TranslationKey0(key: String) extends AbstractCase with TranslationKey {
```

Comment:
I guess you could move `AbstractCase` to `TranslationKey` to avoid repetition

Reply:
If it works, sure. I thought it only reduces code size when used directly.